src/clj/game/core/actions.clj

```diff
@@ -156,18 +156,23 @@
                (or (= last-zone :play-area)
                    (same-side? side (:side card))))
       (let [move-card-to (partial move state s c)
+            card-prompts (filter #(= (get-in % [:card :title]) (:title c) ) (get-in @state [side :prompt]))
```

Comment:
```suggestion
            card-prompts (filter #(same-card? :title % c) (get-in @state [side :prompt]))
```

Reply:
lmao ah shit, i hecked this up. i shouldn't write suggestions before i've had my coffee. don't fix it cuz i'm still undecided on whether the PR is a good idea, but it should be `(same-card? :title (:card %) c)`